colandr/apis/auth.py

```diff
@@ -69,7 +69,9 @@ def post(self, email, password):
         except ValueError:
             return not_found_error("no user found matching given email and password")
         if not user.is_confirmed:
-            return unauthorized_error("user has been created but is not yet confirmed")
+            # TODO: confirm that we want to allow uncomfirmed user logins
+            current_app.logger.warning("%s not yet confirmed")
+            # return unauthorized_error("user has been created but is not yet confirmed")
```

Comment:
@sophiaharms belatedly following back up on this -- is this the behavior you need? if so, do you recommend i put safeguards in for other endpoints to require user confirmation??